### ISSUE TITLE
Peer Review HW 7

### ISSUE THREAD

**Issue by ksedivyhaley** [STAT545-UBC/Discussion#431]
Will we be peer reviewing homework 7 (Make)?

**Comment by jennybc** [comment]
Yes! It's a little trickier to do the assignments, because I need to figure out who did the HW. But thanks for this reminder. I need to try our some changes to the package I use for GitHub API stuff, so this is a splendid excuse to get on that!

**Comment by jennybc** [comment]
This should be done now. So speak up if you participated in HW07 and did NOT see issues re: peer review go by.

**Comment by sinaneza** [comment]
I have participated in HW07 and not assigned for peer review either

**Comment by sinaneza** [comment]
I have participated in HW07 and not assigned for peer review either.
@jennybc

**Comment by jennybc** [comment]
@sinaneza I just tagged you in both peer review threads, so that should create yet another notification. I did include you in the peer review process.
